indexer/packages/postgres/__tests__/stores/wallet-table.test.ts

```diff
@@ -90,48 +90,68 @@ describe('Wallet store', () => {
     expect(wallet).toEqual(expect.objectContaining(defaultWallet2));
   });
 
-  it('Successfully updates totalVolume for time window multiple times', async () => {
-    const firstFillTime = await populateWalletSubaccountFill();
-
-    // Update totalVolume for a time window that covers all fills
-    await WalletTable.updateTotalVolume(
-      firstFillTime.minus({ hours: 1 }).toISO(), // need to minus because left bound is exclusive
-      firstFillTime.plus({ hours: 1 }).toISO(),
-    );
-    let wallet = await WalletTable.findById(defaultWallet.address);
-    expect(wallet).toEqual(expect.objectContaining({
-      ...defaultWallet,
-      totalVolume: '103',
-    }));
-
-    // Update totalVolume for a time window that excludes some fills
-    // For convenience, we will reuse the existing fills data. The total volume calculated in this
-    // window should be added to the total volume above.
-    await WalletTable.updateTotalVolume(
-      firstFillTime.toISO(), // exclusive -> filters out first fill from each subaccount
-      firstFillTime.plus({ minutes: 2 }).toISO(),
-    );
-    wallet = await WalletTable.findById(defaultWallet.address);
-    expect(wallet).toEqual(expect.objectContaining({
-      ...defaultWallet,
-      totalVolume: '105', // 103 + 2
-    }));
-  });
-
-  it('Successfully updates totalVolumeUpdateTime in persistent cache', async () => {
-    const leftBound = DateTime.utc().minus({ hours: 1 });
-    const rightBound = DateTime.utc();
-    await WalletTable.updateTotalVolume(leftBound.toISO(), rightBound.toISO());
-
-    const persistentCache = await PersistentCacheTable.findById('totalVolumeUpdateTime');
-    const lastUpdateTime = persistentCache?.value
-      ? DateTime.fromISO(persistentCache.value)
-      : undefined;
+  describe('Wallet .updateTotalVolume()', () => {
```

Comment:
nit: since top level already denotes that this is for wallet, you can just do 
```suggestion
  describe('updateTotalVolume', () => {
```